### PR TITLE
Update to react-dsfr 0.5.3

### PR DESCRIPTION
**Pull request by desoindx**

### CODE REVIEW

**Comment by sandrica89** [comment]
Focus on tab panel are not positionned properly. This is probably caused by the `padding: 0` I added on it to avoid loosing too much space because there are already many space with successive margins. What do you think ? Should we leave it like that or remove the padding?

Selection_009

**Comment by desoindx** [comment]
> Focus on tab panel are not positionned properly. This is probably caused by the `padding: 0` I added on it to avoid loosing too much space because there are already many space with successive margins. What do you think ? Should we leave it like that or remove the padding?
> 
> Selection_009

Fixed

File: frontend/src/components/PsyListing/PsyTable.jsx [hunk 1]
```diff
@@ -69,54 +108,10 @@ const PsyTable = ({
...
-              <tr key="headers">
-                <th scope="col">Nom</th>
-                <th scope="col">Adresse</th>
-                {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-                <th scope="col" />
-              </tr>
-            </thead>
-            <tbody>
-              {psychologists
-                .slice((page - 1) * 10, page * 10)
```
Comment (sandrica89) [inline frontend/src/components/PsyListing/PsyTable.jsx:83]: No need to "slice" anymore? With `data` prop, does `Table` handle pagination internaly? If yes, how is it linked with the `Pagination` component?

File: frontend/src/App.css [hunk 2]
```diff
@@ -129,5 +129,6 @@
 @media (min-width: 48em) {
   .faqPage .fr-tabs__panel {
     padding: 0; 
+    outline-offset: 0;
```
Comment (sandrica89) [inline frontend/src/App.css:132]: :clap:

File: frontend/src/components/PsyListing/PsyTable.jsx (hunk 1 above)
Comment (desoindx) [reply frontend/src/components/PsyListing/PsyTable.jsx:83]: good catch !

**Review by sandrica89 — APPROVED** [review]

**Pull request merged**